<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMBISTDlg</class>
 <widget class="QDialog" name="IMBISTDlg">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imec Diagnostics</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Help: &quot;?&quot;</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="testCB">
      <item>
       <property name="text">
        <string>Test 4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test 5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test 6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test 7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test 8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test 9</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="goBut">
      <property name="text">
       <string>Go</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="outTE">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>361</width>
     <height>181</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBut">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBut">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>testCB</tabstop>
  <tabstop>goBut</tabstop>
  <tabstop>outTE</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
